priv/repo/migrations/20210216142615_add_vehicle_adherence_color_to_user_setting.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Skate.Repo.Migrations.AddVehicleAdherenceColorToUserSetting do
+  use Ecto.Migration
+
+  def change do
+    execute(
+      "CREATE TYPE vehicle_adherence_colors AS ENUM ('early_red', 'early_blue')",
+      "DROP TYPE vehicle_adherence_colors"
+    )
+
+    alter table(:user_settings) do
+      add(:vehicle_adherence_colors, :vehicle_adherence_colors, default: "early_red")
```

Comment:
One thing--I don't believe that setting `default` backfills existing rows (although I haven't checked so I could be wrong), so I suggest adding another `execute` that does `UPDATE user_settings SET vehicle_adherence_colors='early_read'`.

Reply:
So interestingly I saw that you did the `execute` by hand here but when I tested locally it didn't seem like I needed it. That said now that I type this out I forget whether or not my local version of Postgres at all matches what Skate is using in prod, so I'll add it in just to be safe.

Reply:
Maybe `default` _does_ backfill, I'll look into that sometime.